{
  "ID": "oval:org.debian:def:201810879",
  "Class": "vulnerability",
  "Metadata": {
    "Title": "CVE-2018-10879",
    "AffectedList": [
      {
        "Family": "unix",
        "Platform": "Debian GNU/Linux 9",
        "Product": "linux"
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2018-10879",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-10879"
      }
    ],
    "Description": "A flaw was found in the Linux kernel's ext4 filesystem. A local user can cause a use-after-free in ext4_xattr_set_entry function and a denial of service or unspecified other impact may occur by renaming a file in a crafted ext4 filesystem image.",
    "Debian": {
      "MoreInfo": ""
    }
  },
  "Criteria": {
    "Operator": "AND",
    "Criterias": [
      {
        "Operator": "OR",
        "Criterias": [
          {
            "Operator": "AND",
            "Criterias": null,
            "Criterions": [
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:2",
                "Comment": "all architecture"
              },
              {
                "Negate": false,
                "TestRef": "oval:org.debian.oval:tst:16770",
                "Comment": "linux DPKG is earlier than 4.9.110-1"
              }
            ]
          }
        ],
        "Criterions": null
      }
    ],
    "Criterions": [
      {
        "Negate": false,
        "TestRef": "oval:org.debian.oval:tst:1",
        "Comment": "Debian 9 is installed"
      }
    ]
  }
}